python/utils/OpaqueArguments.h

```diff
@@ -185,18 +185,18 @@ inline std::string mlirTypeToString(mlir::Type ty) {
   return msg;
 }
 
-inline void
-packArgs(OpaqueArguments &argData, py::args args,
-         mlir::func::FuncOp kernelFuncOp,
-         const std::function<bool(OpaqueArguments &argData, py::object &arg)>
-             &backupHandler) {
+inline void packArgs(OpaqueArguments &argData, py::args args,
+                     mlir::func::FuncOp kernelFuncOp,
+                     const std::function<bool(OpaqueArguments &argData,
+                                              py::object &arg)> &backupHandler,
+                     std::size_t startingArgIdx = 0) {
```

Comment:
No, that is how the formatter is choosing to format it. It probably looks a little strange because of the `bool(...args...)` inside the `std::function<>` brackets, but I think it is formatting it as it intends to.